apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -832,3 +834,7 @@ func clusterTemplateTypes(schemas *types.Schemas) *types.Schemas {
 		MustImport(&Version, v3.ClusterTemplate{}).
 		MustImport(&Version, v3.ClusterTemplateRevision{})
 }
+
+func clusterScanTypes(schemas *types.Schemas) *types.Schemas {
+	return schemas.MustImport(&Version, v3.ClusterScan{})
```

Comment:
we should disable create and update (PUT/POST) from the schema as it will be created internally only